Snowdrift.cabal

```diff
@@ -215,9 +215,9 @@ library
                  , time
                  , titlecase
                  , transformers
-                 , unix
```

Comment:
You should use an `if`-block here, and set the dependency to `unix-compat` for Windows. I'm not certain, but I think unix is a hard dependency of Snowdrift.